jsonDash/webHook/views.py

```diff
@@ -0,0 +1,64 @@
+from .parser import json_parser
+from .models import jsonNode
+from rest_framework import viewsets
+from rest_framework.response import Response
+from rest_framework import status
+import logging
+from .models import jsonNode
+from .serializers import Node_Serializer
+# Create your views here.
+class Node_API_View(viewsets.ModelViewSet):
+
+    queryset = jsonNode.objects.all()
+    serializer_class = Node_Serializer
+
+    def list(self, request, *args, **kwargs):
+        logging.info(f"{request} - list all objects")
+        data = list(jsonNode.objects.values())
+        return Response(data,status=status.HTTP_200_OK)
+    
+    def retrieve(self, request, *args, **kwargs):
+        logging.info(f'{request},find - id = {kwargs["pk"]}')
+        try:
+            data = jsonNode.objects.filter(id = kwargs['pk'])[0]
+            serializer = Node_Serializer(data)
+            logging.info('Object found')
+            return Response(serializer.data,status = status.HTTP_200_OK)
+        except jsonNode.DoesNotExist:
+            logging.exception('Object not found')
+            return Response(status=status.HTTP_404_NOT_FOUND)
+        except IndexError:
+            logging.exception("Index out of range")
+            return Response(status=status.HTTP_404_NOT_FOUND)
+        
+    def create(self,request,*args,**kwargs):
+        logging.info(f"{request},{request.data}")
```

Comment:
```suggestion
        logging.info(f"{request}, {request.data}")
```

Прогони код через pep8